doc/doxygen/headers/glossary.h

```diff
@@ -346,6 +346,43 @@
  * VectorTools::interpolate_boundary_values,
  * VectorTools::compute_no_normal_flux_constraints.
  *
+ * It is often useful to visualize the boundary indicators used on different
+ * parts of the boundary of a domain. For example, when reading in a complex
+ * mesh generated by an external program, one typically wants to verify
+ * that what is *believed* to be the assignment of boundary indicators is
+ * indeed what is in the mesh file, or maybe equally importantly, what deal.II
+ * has read in from the mesh file. This can be done by using the
+ * DataPostprocessors::BoundaryIds class and the following small piece of code:
+ * @code
+ *    DataPostprocessors::BoundaryIds<dim> boundary_ids;
+ *    DataOutFaces<dim> data_out_faces;
+ *    FE_Q<dim>         dummy_fe(1);
+ *
+ *    DoFHandler<dim>   dummy_dof_handler(triangulation);
+ *    dummy_dof_handler.distribute_dofs(dummy_fe);
+ *
+ *    Vector<double> dummy_solution (dummy_dof_handler.n_dofs());
+ *
+ *    data_out_faces.attach_dof_handler(dummy_dof_handler);
+ *    data_out_faces.add_data_vector(dummy_solution, boundary_ids);
```

Comment:
I have not tried this myself, but can't you just set
```c++
  data_out_faces.attach_triangulation(triangulation);
  data_out_faces.build_patches();
```
It might be that you still need to pass a dummy vector (of cell values). At least for `DataOut`, we worked towards not needing to set a `DoFHandler` in case we only write cell data, see also the documentation of the function:
https://github.com/dealii/dealii/blob/b1b07cec07e64dcb0424ae809ba8cf3b3dd86560/include/deal.II/numerics/data_out_dof_data.h#L654-L664
And in case this does not work, I think it should be a rather limited step to actually remove the need for `DoFHandler` also in `DataOutFaces`, and what I would prefer over teaching to create dummy values.

Reply:
I am not 100% but I think DataOut also requires a DoFHandler if you want to use a DataPostProcessor.

Reply:
@tjhei is right: When using a postprocessor, we need a `dof_handler` object:
```
template <int dim, int patch_dim, int spacedim, int patch_spacedim>
template <typename VectorType>
void
DataOut_DoFData<dim, patch_dim, spacedim, patch_spacedim>::add_data_vector(
  const VectorType &                 vec,
  const DataPostprocessor<spacedim> &data_postprocessor)
{
  Assert(dofs != nullptr,
         Exceptions::DataOutImplementation::ExcNoDoFHandlerSelected());
  add_data_vector(*dofs, vec, data_postprocessor);
}
```
So that won't work.